charon/Cargo.toml

```diff
@@ -69,7 +69,8 @@ stacker = "0.1"
 take_mut = "0.2.2"
 toml = { version = "0.8", features = ["parse"] }
 tracing-subscriber = { version = "0.3", features = [ "env-filter", "std", "fmt" ] }
-tracing-tree = "0.3"
+tracing-tree = { git = "https://github.com/Nadrieril/tracing-tree", features = [ "time" ] } # Fork with improved formating and timing info.
```

Comment:
What is your plan for this fork?
If we keep this change, on the longer term I think we should make the fork live in the AeneasVerif repo.
What do you think?

Reply:
The goal would be to upstream my changes to the main `tracing-tree` project. I'll experiment more in charon first though.

Importantly, note that this is a drop-in replacement for the upstream version. So we could revert to the upstream version and only use my fork locally when debugging, if using my fork like this is undesireable.

Reply:
Upstreaming the changes to the main `tracing-tree` is probably the best thing to do.